frontend/src/styles/base/colors.scss

```
@@ -3,3 +3,4 @@
 /////////////////////////
 
 $shadow: rgba(0, 82, 174, .15);
+$pinpoint-primary: #EA0808;
```

nitpick - I think it's safe to just call this `primary`:

```suggestion
$primary: #EA0808;
```